app/sheet_fetcher.py

```diff
@@ -0,0 +1,44 @@
+import configparser
+
+import gspread as gspread
+
+from app import db
+from app.models import OckovaciMisto
+
+
+class SheetFetcher:
+    SHEET_API = 'https://onemocneni-aktualne.mzcr.cz/api/v2/covid-19/prehled-ockovacich-mist.json'
```

Comment:
Maybe these constants should be all in one place ?

Reply:
I would leave it for improvements, count of days should be also somewhere else, but it has to be reflected also in GUI